### ISSUE TITLE
CUDNN 8.6 Install appears to be broken

### ISSUE THREAD

**Issue by Derpiesaurus** [devilismyfriend/StableTuner#88]
It may just be my computer/network connection, but the automatic CUDNN 8.6 installation doesn't work, it says it takes too long to respond and errors.

I got around this by downloading the .ZIP file from NVIDIA's website for the Windows version of CUDNN 8.6, unzipped it, renamed it to "cudnn_windows", and put it in the "resources" folder.

**Comment by KewkLW** [comment]
Make an account and download it direct if the host is having issues https://developer.nvidia.com/cudnn I'm using 8.7 as there's a huge performance increase with it.

**Comment by Derpiesaurus** [comment]
> Make an account and download it direct if the host is having issues https://developer.nvidia.com/cudnn I'm using 8.7 as there's a huge performance increase with it.

Yep that's what I ended up doing. I didn't get 8.7 though, so I'll definitely look into that.
